apps/hexagon_matmul/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../support/Makefile.inc
```

Comment:
That refactoring already happened for most apps. Copy-paste from local laplacian or bilateral grid instead. Which ones still need fixing?